easybuild/easyconfigs/f/fastQValidator/fastQValidator-0.1.1a-dev-goolf-1.7.20-aadc6f9.eb

```diff
@@ -0,0 +1,44 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'fastQValidator'
+version = '0.1.1a-dev'
+fastqvalidator_git_commit = 'aadc6f9'
+libstatgen_git_commit = '8246906'
+versionsuffix = '-%s' % fastqvalidator_git_commit
+
+homepage = 'http://genome.sph.umich.edu/wiki/FastQValidator'
+description = """Validate FastQ Files"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+# we download latest source available in github for fastQValidator and libStatGen (dependency)
+# using latest release tags doesn't work
+source_urls = ['https://github.com/statgen/fastQValidator/archive/', 'https://github.com/statgen/libStatGen/archive/']
+sources = ['%s.tar.gz' % fastqvalidator_git_commit, '%s.tar.gz' % libstatgen_git_commit]
+
+dependencies = [
+    ('zlib', '1.2.8'),
+#   ('OpenSSL', '1.0.1q'),  # OS dependency should be preferred if the os version is more recent then this version, it's
+#   nice to have an up to date openssl for security reasons
```

Comment:
this dep is required by libstatgen. You are right I forgot the osdep, I will add it